hopper/src/main/java/org/atomhopper/abdera/parser/EntryJsonPOJO.java

```diff
@@ -0,0 +1,285 @@
+package org.atomhopper.abdera.parser;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.abdera.Abdera;
+import org.apache.abdera.factory.Factory;
+import org.apache.abdera.model.Element;
+
+import javax.xml.namespace.QName;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class EntryJsonPOJO {
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class Root {
+
+        private EntryWrapper entry;
+
+        public EntryWrapper getEntry() {
+            return entry;
+        }
+
+        public void setEntry(EntryWrapper entry_data) {
+            this.entry = entry_data;
+        }
+    }
```

Comment:
You're effectively treating this `Root` object as the opening `{` in the JSON. Instead, treat the class as the Root instead of this nested class definition.